### PR TITLE
Add news post for Playtest 20190825

### PR DESCRIPTION
**Pull request by pchote**
Screenshot_2019-08-24 OpenRA

Screenshot 2019-08-24 at 21 10 48

### CODE REVIEW

**Comment by reaperrr** [comment]
Didn't notice any issues apart from the TODOs.

**Comment by pchote** [comment]
Finished TODOs.

File: content/news/playtest-20190825.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+On the gameplay front, the main focus for this release was to rewrite the logic that controls how units behave and respond to orders. This was a major task that spanned several months, more than 90 Pull Requests, and more than one debate about bug vs "accidental feature".
+
+Many of the changes will not be immediately obvious, but on closer inspection you should find that many long-standing quirks and annoyances in unit behaviour no longer occur.
+
+The ability to queue commands using the Shift key has always been an important part of OpenRA's gameplay, and this has received a major overhaul with this release. Part of the overhaul has moved the familiar targeting lines that display when you command units into the "Waypoint Mode" (Shift) overlay by default. The previous behaviour can be restored through the Settings menu (Target Lines: Automatic) if you prefer.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190825-waypoint.png" alt="Waypoint mode." width="600px">
+
+The improved waypoint mode allows you to plan your attack strategies
```
Comment (teinarss) [inline content/news/playtest-20190825.md]: Missind . at the end?

**Comment by pchote** [comment]
Fixed.

**Comment by anjew175** [comment]
Do you think we should use the 'about-highlight' css class instead of 'color: #d92c2c' to highlight the words in the newspost consistently with how the site is now? Right now every page except the front page is using a yellow highlight after #453

**Pull request merged**